src/Pages/Runways/components/Card.scss

```
@@ -0,0 +1,23 @@
+.Card {
+  .container {
+    margin-right: 10px;
+    div {
+      width: 320px;
+      height: 760px;
+      overflow: hidden;
+      img {
+        height: 100%;
```

```suggestion
  .container {
    margin-right: 10px;
    
    div {
      width: 320px;
      height: 760px;
      overflow: hidden;
      
      img {
        height: 100%;
```
새로운 셀렉터 시작하는 자리에서는 한줄씩 띄워주세요!